balena/models/device_os.py

```diff
@@ -407,16 +491,17 @@ def __transform_host_apps(self, host_apps):
         os_versions_by_device_type = {}
 
         for host_app in host_apps:
-            host_app_device_type = host_app['is_for__device_type'][0]['slug'] if len(host_app['is_for__device_type']) > 0 else None
+            if host_app['is_host']:
```

Comment:
`__get_os_versions` already has a $filter with `is_host: true`, so I think we shouldn't need this, right?